server/services/hubTags.js

```diff
@@ -33,35 +33,8 @@ const createHubTagsService = repository => {
     }
   }
 
-  function relatedSeriesFor({ id, establishmentId, perPage, offset }) {
-    return repository.seasonFor({
-      id,
-      establishmentId,
-      perPage,
-      offset,
-    });
-  }
-
-  function relatedContentFor({
-    id,
-    establishmentId,
-    perPage,
-    offset,
-    sortOrder,
-  }) {
-    return repository.relatedContentFor({
```

Comment:
is `repository.relatedContentFor` used elsewhere?

Reply:
and relatedSeries?

Reply:
So `repository.relatedContentFor` is used on line 21 still; assuming that route is still valid 🤔 
If you meant `relatedSeriesFor`, it has been completely removed.
I guess you maybe meant `repository.seasonFor`, but that still used on line 11.
To be honest allot of this seems rather unpleasant to unpick; needless complexity 🤷‍♀️